public/data/Aryan.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Aryan Srivastava",
+  "bio": "Android and open source enthusiast",
+  "avatar": "https://github.com/Aryan-Srivastava.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "http://github.com/FayasNoushad",
```

Comment:
- Please Update your github Url

Reply:
Change that link to `https://github.com/Aryan-Srivastava`.

Reply:
Thanks for telling.